src/components/slide-toggle/slide-toggle.ts

```diff
@@ -59,6 +61,12 @@ export class MdSlideToggle implements AfterContentInit, ControlValueAccessor {
   private _isMousedown: boolean = false;
   private _isInitialized: boolean = false;
 
+  // Thumb Container Element, which is required for the dragging functionality.
+  @ViewChild('thumbContainer') private _thumbContainerRef: ElementRef;
+
+  // Pointer Object for the dragging functionality of the thumb.
+  private _dragPointer: any;

```

Comment:
This should have a real type. I also don't quite follow what it's meant to encompass.